android/src/fdroid/play/listings/de-DE/full-description.txt

```diff
@@ -1,52 +1,52 @@
-‣ Our free app does not track you and does not have ads, and is driven by your donations.
-‣ It is constantly being improved by our small team, in our free time, using our own money and donations/contributions from our users.
-‣ If something is wrong or missing on the map, please fix it in <b>OpenStreetMap</b> and see your changes in the future maps update.
-‣ If navigation or search doesn't work, please email us. We reply to <i>EVERY</i> email, and we'll fix it ASAP!
-
-<b>Your feedback and 5-star reviews are the best motivators for us!</b>
-
-Key features:
-• Free, open-source, no ads, no tracking
-• Detailed offline maps with places that don't exist on Google maps, thanks to the <b>OpenStreetMap</b> community
-• Cycling routes, hiking trails, and walking paths
-• Contour lines, elevation profiles, peaks, and slopes
-• Turn-by-turn walking, cycling, and EXPERIMENTAL car navigation with voice guidance
-• Fast offline search
-• Bookmarks export and import in KML/KMZ formats (GPX is coming soon)
-• Dark mode to protect your eyes
-
-There is no Android Auto, public transport, satellite maps, and other cool features <i>yet</i> in Organic Maps. But with <i>your help and support</i>, we can make this world better step by step.
-
-Organic Maps is <b>pure and organic, made with love</b>:
-
-• Blazing fast offline experience
-• Respects your privacy
-• Saves your battery
-• No unexpected mobile data charges
-• Simple to use, with only most important features included
-
-Free from trackers and other bad stuff:
-
-• No ads
-• No tracking
-• No data collection
-• No phoning home
-• No annoying registration
-• No mandatory tutorials
-• No noisy email spam
-• No push notifications
-• No crapware
+‣ Unsere kostenlose App trackt Sie nicht, enthält keine Werbung und wird durch Ihre Spenden finanziert.
+‣ Sie wird laufend von unserem kleinen Team, in unserer Freizeit, mit unserem eigenen Geld und Spenden/Beiträgen unserer Nutzer, verbessert.
+‣ Wenn etwas auf der Karte falsch ist oder fehlt, korrigieren Sie es bitte in <b>OpenStreetMap</b> und sehen Sie Ihre Änderungen in zukünftigen Karten-Updates.
+‣ Sollte die Navigation oder die Suche nicht funktionieren, senden Sie uns bitte eine E-Mail. Wir antworten auf <i>JEDE</i> E-Mail und werden das Problem so schnell wie möglich beheben!
+
+<b>Ihr Feedback und Ihre 5-Sterne-Bewertungen sind die beste Motivation für uns!</b>
+
+Wichtigsten Eigenschaften:
+• Kostenlos, Open-Source, keine Werbung, kein Tracking
+• Detaillierte Offline-Karten mit Orten, die es auf Google Maps nicht gibt, und zwar dank der <b>OpenStreetMap</b>-Community
+• Radwege, Wanderwege und Spazierwege
+• Höhenlinien, Höhenprofile, Gipfel und Steigungen
+• Turn-by-Turn-Navigation für Fußgänger, Radfahrer und EXPERIMENTELLE Auto-Navigation mit Sprachführung
```

Comment:
"Turn-by-Turn-Navigation" could be called "Abbiegehinweise"

Reply:
I thought this is a used term. At least there is a german Wiki article: https://de.wikipedia.org/wiki/Turn-by-Turn-Navigation .

For me "Abbiegehinweise" is too unprecise. Maybe "Navigation mit Abbiegehinweisen"?

Reply:
I don't think Turn-by-Turn-Navigation is very widespread. Especially for people who don't speak English, it would be almost impossible to know what it means.

> For me "Abbiegehinweise" is too unprecise. Maybe "Navigation mit Abbiegehinweisen"?

Yes that's better.